### DISCUSSION TITLE
Upload not working when Google Tag Manager is included

### DISCUSSION THREAD

**Discussion by wilsonseixo** [livewire/livewire#5675] [1. Help] [answered]
Hey everyone,

A weird problem just came up. We have a component to upload files using `WithFileUploads`.

Now, when we added Google Tag Manager script to the view, the upload stops working without giving up any error.

We are using version `2.10`.

Any thoughts?

**Comment by wilsonseixo** [comment]
Anyone with this issue?

**Reply by Miguel-D20** [reply]
I have the same issue and I don't find any post or discussion talking about this anywhere :/

**Reply by technik-cw** [reply]
We had this problem as well and disabling form Interactions in GA4 fixed it. Currently we are troubleshooting how we can enable and use it again.

<img width="742" alt="image" src="https://github.com/livewire/livewire/assets/68112434/664bf1ab-c808-40f4-b8b1-48200bfe7c9f">

**Reply by kent013** [reply]
I have same issue and turning off "Form interactions" solves problem.
Thanks @technik-cw

**Comment by Miguel-D20** [comment, marked as answer]
The solution that @technik-cw provided works. For anyone with the same issue, follow these steps:

1. Using the GA4 Admin Panel:
2. Log in to your Google Analytics account and navigate to the Admin section.
3. Under "Data collection and modification," click on "Data streams."
4. Select the data stream you want to modify (usually your website).
5. Locate the "Enhanced Measurement" section. You might see a gear icon next to it.
6. Click on the gear icon or the "Edit" option to view individual settings.
7. Find the toggle switch for "Form interactions."
8. Slide the switch to "Off" to disable tracking form starts and submissions.
9. Click "Save" to apply the changes.

![Step 1](https://github.com/livewire/livewire/assets/81695799/d9c3908f-4ba6-43ea-8eb4-c00fd044a3a5)
![Step 2](https://github.com/livewire/livewire/assets/81695799/c34adf10-bef2-4825-8a12-f989d98f5596)
![Step 3](https://github.com/livewire/livewire/assets/81695799/e6a5893d-d452-479a-8bde-e34a76d0aed4)
